src/app/views/common/monaco/MonacoV9.tsx

```diff
@@ -0,0 +1,79 @@
+import { makeStyles } from '@fluentui/react-components';
+import { Editor, OnChange } from '@monaco-editor/react';
+import { editor } from 'monaco-editor';
+import { ThemeContext } from '../../../../themes/theme-context';
+import { formatJsonStringForAllBrowsers } from './util/format-json';
+import { useAppSelector } from '../../../../store';
+import { Mode } from '../../../../types/enums';
+
+interface MonacoProps {
+  body: object | string | undefined;
+  onChange?: OnChange;
+  verb?: string;
+  language?: string;
+  readOnly?: boolean;
+  height?: string;
+  extraInfoElement?: JSX.Element;
+}
+
+const useEditorStyles = makeStyles({
+  container: {
+    height: '95%'
```

Comment:
I'm getting these overlays on light and dark theme mode. Adding `position: 'relative',
    zIndex: -1` to these container styles fixes it for me.
Also in monaco-editor.css
![screenshot-1739434014329](https://github.com/user-attachments/assets/1713f873-44f3-4469-b658-df40f7cf2fc0)
![screenshot-1739433991438](https://github.com/user-attachments/assets/6fd90826-2f3b-4005-af5f-b9f8349cd98f)

Reply:
zIndex of -1 will just move it back but it won't prevent it from overflowing. The ideal solution is to have monaco adjusting to the height of the request/response area.

Reply:
Moving this to a PR of it's own to allow more exploration.